functions/Get-DbaAgentJob.ps1

```diff
@@ -52,6 +60,8 @@ FUNCTION Get-DbaAgentJob {
 		$SqlCredential,
 		[object[]]$Job,
 		[object[]]$ExcludeJob,
+		[ValidateSet('True', 'False')]
+		[string]$Enabled,
```

Comment:
I like that idea better, didn't think of that. I'll get an update together, thanks for the review!